### PR TITLE
Fixed broken links - 2023.03

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR fixes broken links in the documentation.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement develoiper
NONE
```

### CODE REVIEW

File: website/blog/2021/01.25-Machine-Controller-Manager.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Metal-stack is a set of microservices that implements Metal as a Service (MaaS).
...
 
-Also, other interesting use cases with MCM are implemented by Kubernetes enthusiasts, who for example adjusted the Machine Controller Manager to provision machines in the cloud to extend a local Raspberry-Pi K3s cluster. Read more about it in [Scale out your Raspberry-Pi Kubernetes cluster to the cloud](https://nativecloud.dev/scale-out-your-raspberry-pi-k3s-cluster-to-the-cloud/) or check out [this video](https://youtu.be/UuveyEOn4_o?t=60) on our [Gardener Project](https://www.youtube.com/channel/UCwUhwKFREV8Su0gwAJQX7tw) YouTube channel.
+Also, other interesting use cases with MCM are implemented by Kubernetes enthusiasts, who for example adjusted the Machine Controller Manager to provision machines in the cloud to extend a local Raspberry-Pi K3s cluster. For more information, you can check out [this video](https://youtu.be/UuveyEOn4_o?t=60) on our [Gardener Project](https://www.youtube.com/channel/UCwUhwKFREV8Su0gwAJQX7tw) YouTube channel.
```
Comment (JordanJordanov) [inline website/blog/2021/01.25-Machine-Controller-Manager.md:98]: Shall we replace "this video" with the video name, in this case "Gardener Community Meeting 2020-07-03"?

**Review by JordanJordanov — APPROVED** [review]
/lgtm

**Comment by JordanJordanov** [comment]
CC @n-boshnakov , @Kostov6 
After [rerun ](https://concourse.ci.gardener.cloud/teams/gardener/pipelines/documentation-master/jobs/master-pull-request-job/builds/9.1) the `concourse-ci/check-manifest-entrypoint` check failed again. Is it catching a valid issue with the manifest?

**Comment by Kostov6** [comment]
https://github.com/gardener/machine-controller-manager/pull/797/files introduces unparsable URL's that cause the break even in the [website ci/cd](https://concourse.ci.gardener.cloud/teams/gardener/pipelines/gardener-website-generator-master/jobs/master-head-update-job/builds/61)

**Comment by JordanJordanov** [comment]
The new line [here](https://github.com/gardener/machine-controller-manager/pull/797/files#diff-705d16133827dd79e2b846d5fc9ce28d54335825287f7be2dc5ddb8047445b11R5961-R5962)?

**Comment by n-boshnakov** [comment]
I have [fixed the broken link](https://github.com/gardener/machine-controller-manager/pull/799) in the machine-controller-manager repo, waiting for the PR to be merged.

**Comment by Kostov6** [comment]
This document is generated automatically and [this PR](https://github.com/gardener/machine-controller-manager/pull/800) fixes a problem with the generation

**Pull request merged**